### PR TITLE
Milestone 1: Create a new repo to review a recent project

### PR DESCRIPTION
**Pull request by carloshs1994**
## In this milestone:

-  Copied the code from a recent project from the trials into this new repo to perform a review
- Added linters to the project to make sure looks good.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (carloshs1994) [inline index.html:7]: You may want to put a descriptive title for your project. I know you can come up with creative ideas 🥇

File: index.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main id="main">
+        <H1 id="title">"Juanga" is alive in all of us</H1>
+        <p>El divo de Juares</p>
+        <figure id="img-div">
+            <img id="image" src="Juanga.jpeg" alt="Juan Gabriel with open arms">
```
Comment (carloshs1994) [inline index.html:15]: You may want to create a folder for the Images to improve the quality of the project. You can do it using 
`mkdir images` on the terminal or using VSCode.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main id="main">
+        <H1 id="title">"Juanga" is alive in all of us</H1>
+        <p>El divo de Juares</p>
+        <figure id="img-div">
+            <img id="image" src="Juanga.jpeg" alt="Juan Gabriel with open arms"> 
+            <figcaption id="img-caption">Juan Gabriel in Foro Sol</figcaption>
+        </figure>
```
Comment (carloshs1994) [inline index.html:17]: You may consider using the `<section>` tag to wrap these elements. Your code will look cleaner, that´s always a good thing!

**Review by carloshs1994 — COMMENTED** [review]
## Hi @carloshs1994,

You've done an amazing job so far, I'd like to highlight the following:

PR summary is descriptive
GitHub flow was correctly used
Linter has no errors
Issues need to be resolved before your approval, but you're almost there!

## Required Changes ♻️

Check the comments below!

**Comment by ShoiraTa** [comment]
## Status: Approved :heavy_check_mark:

Hi @carloshs1994 ,  

Your project is complete! Congratulations!  🎉💯🌟 

## Good Points 👍 
- Being polite with suggestions  ✔️ 
- Explaining the reason for requested changes  ✔️ 
- Comments are clear ✔️ 




Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### [OPTIONAL]: 

- [x] I kindly suggest you to use at least 1 emoji for each comment to make comments human-friendly  🤖 😁 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by carloshs1994** [comment]
## **Thanks, @ShoiraTa!!!** 
Love the suggestion and I will definitely do it 💯 😄

**Comment by carloshs1994** [comment]
BTW @ShoiraTa  I´ve just sent you my GitHub repo as feedback so don´t laugh too hard when you read it 🤣 🤣 🤣

**Pull request merged**